website_v2/gcloud-powershell-website-v2/www/static/app/mainController.js

```diff
@@ -0,0 +1,12 @@
+var app = angular.module('powershellSite');
+
+app.controller('MainController',
+    function($scope, $route, $routeParams, $location, $http) {
+    $scope.$route = $route;
+    $scope.$location = $location;
+    $scope.$routeParams = $routeParams;
+    $http.get('static/_data/cmdletsFull.json')
+       .then(function(res) {
+           $scope.products = res.data;
```

Comment:
If we do set a value on `$scope` inside of a function, @iantalarico would you provide a default value outside of that function? e.g. in this case set `$scope.products = null;` above on line 8 or so?